functions/src/modules/pm/categories/deleteCategory.ts

```diff
@@ -1,5 +1,27 @@
-import { ExpressFunction } from "../../../@types";
+import type { ExpressFunction } from "../../../@types";
+import { firebaseApp } from "../../../config/firebaseConfig";
+import { expressErrorHandlerFactory } from "../../../helpers";
+import { uasPermissionSwitch } from "../../../systems/uas";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
+const deleteCategory: ExpressFunction = (req, res, next) => {
+  uasPermissionSwitch({
+    organizer: { accepted: execute },
+  })(req, res, next);
+};
 
-export default replaceMe;
+const execute: ExpressFunction = (req, res, next) => {
+  const errorHandler = expressErrorHandlerFactory(req, res, next);
+
+  firebaseApp
+    .firestore()
+    .collection("prizeCategories")
+    .doc(req.params.companyId)
```

Comment:
I think this should be `categoryId`, right?
```suggestion
    .doc(req.params.categoryId)
```